Fix precedence of the quanta layer by adding a base layer for all the… (#6539)
plone/volto: packages/components/src/stories/Introduction.mdx

# `@plone/components`

This package contains ReactJS components for Plone 7 and Plone's API-first CMS story.

The purpose of this package is to provide an agnostic set of baseline components to build Plone sites upon.

## Usage

> Using the package manager of your choice (npm, yarn, pnpm) to install the package in your app.

Reviewer: ```suggestion
You can use your choice of package manager (npm, yarn, or pnpm) to install the package in your app.
```

> Use pnpm in case you are adding them a Volto add-on/workspace:

Reviewer: ```suggestion
If you add the components to a Volto add-on or workspace, use pnpm, as shown.
```

import '@plone/components/src/styles/basic/TextField.css';
```

### CSS layers

> This package use CSS layers to provide a way to style the components in a more flexible way.

Reviewer: ```suggestion
This package uses CSS layers to style the components in a more flexible way.
```

It uses the `plone-components` layer name to scope all the CSS declarations in the package.
The basic styling uses the nested `plone-components.base` named layer.
You can use the `plone-components` layer to override the basic styling, or use the `plone-components.base` layer to override the basic styling in a more specific way.

> ## Quanta

Reviewer: I think this needs to increase a level because it is under Styling, not a sibling to it.

```suggestion
### Quanta
```

This package also features the Quanta components.

> These components use the basic styling as a baseline, extending them to achieve Quanta look and feel.

Reviewer: ```suggestion
These components use the basic styling as a baseline, extending them to achieve the Quanta look and feel.
```
